schema_generator/search.py

```diff
@@ -77,30 +78,44 @@ def _derived_options_transformation_generator(checkbox_question):
 }
 
 
-def get_transformations(framework_slug):
-    for question in _get_questions_by_type(framework_slug, TRANSFORMATION_GENERATORS.keys()):
+def get_transformations(framework_slug, doc_type):
+    for question in _get_questions_by_type(framework_slug, doc_type, TRANSFORMATION_GENERATORS.keys()):
         for transformer in TRANSFORMATION_GENERATORS[question.type](question):
             yield transformer
 
 
-def generate_search_mapping(framework_slug, file_handle, mapping_type, extra_meta={}):
-    with open(os.path.join(_base_dir, 'frameworks', framework_slug, 'search_mapping.json'), 'r') as h_template:
+def generate_search_mapping(framework_slug, doc_type, file_handle, mapping_type, extra_meta={}):
+    with open(os.path.join(
+        _base_dir,
+        "frameworks",
+        framework_slug,
+        "search_mappings",
+        "{}.json".format(doc_type),
+    ), 'r') as h_template:
```

Comment:
what does the `h` stand for?

Reply:
"handle". Probably naughty of me to sneak in some rotten old hungarian notation, but it's there now and I got away with it so :-p

Reply:
do i spy an ex-windows programmer...?

Reply:
We all have a past, right... (to be fair I never did much C, so don't ask me about that)